### ISSUE TITLE
Wrong results with simple test

### ISSUE THREAD

**Issue by arnaudPatatam** [NathanUA/U-2-Net#39]
Hello,

By doing tests, I created a model with only 1 image and 10 epoq and I tested the model with this same image, to get bad results in the end. Do you have any explanations?

Thank you

**Comment by NathanUA** [comment]
Could you please show us the image, ground truth and the prediction, as well as the training loss ? If you prediction has very corase shape of the target or even just a circular or elliptic gray region. My initial guess is the that 10 epochs are not enough. Try to increase the iteration numbers.

**Comment by arnaudPatatam** [comment]
I no longer have the results, but maybe this test could not just work, but now my model is fine. Thank you
